setup.cfg

```diff
@@ -28,6 +28,8 @@ ignore =
     G200, G202,
     # pycodestyle checks ignored in the default configuration
```

Comment:
unrelated note: there's `extend-ignore` in flake8, it should be used to avoid copy-pasting the defaults here. https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-extend-ignore

Reply:
PRs welcome, in case someone can get to this before me. :)